packages/template-chakra-storefront/src/hooks/use-wish-list.js

```diff
@@ -35,16 +61,137 @@ export const useWishList = ({listId = ''} = {}) => {
         if (!productLists || !isProductListsSuccess || productLists.total) return
```

Comment:
NIT: This is olde code, but no requirement to clean it up. But wouldn't it be nicer to have a default value of an empty array then we don't have to worry about all this additional conditional stuff. Maybe a future enhancement if it makes sense.